src/node-red/index.njk

```diff
@@ -2,79 +2,46 @@
 layout: page
 nohero: true
 meta:
-    title: Node-RED Resources
+    title: Node-RED online for free
```

Comment:
It's not for free?

Reply:
How do you think we should phrase it? Just Node-RED online?

Reply:
"Cloud hosted Node-RED"? Online I don't believe is nice either -- My laptop is online, but it's not an environment I want to run Node-RED for professional use-cases.

Reply:
```suggestion
    title: Start using Node-RED online for free
```